packages/datagateway-search/src/table/datasetSearchTable.component.tsx

```diff
@@ -193,6 +321,10 @@ const mapDispatchToProps = (
               id: { in: luceneData },
             }),
           },
+          {
+            filterType: 'include',
+            filterValue: '"investigation"',
```

Comment:
We also need to include investigationInstrument and Instrument otherwise there's not enough info to form the links. This should instead be `filterValue: JSON.stringify({investigation: { investigationInstruments: 'instrument' }})`